src/Chapter1.hs

```diff
@@ -71,6 +71,7 @@ Modules should have the same name as the corresponding file with
 the `.hs` extension.
 -}
 module Chapter1 where
+import Data.Bits (Bits(xor))
```

Comment:
Wow, interesting start!

I think though that it is not used in the end, so it is safe to remove the import 🙂